doc/usage.rst

```diff
@@ -211,6 +211,11 @@ Then, add the following command to your CI:
 
 Where ``$COMMIT_RANGE`` is your range of commits, e.g. ``$TRAVIS_COMMIT_RANGE`` or ``HEAD~..HEAD``.
 
+### GitLab Code Quality Integration
```

Comment:
This is not Markdown, but Restructured Text, so this style of heading does not work. According to [the docs](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections) it must be "underlined" with different character than in previous headings (`=` and `-` already used), so another nesting level is determined automatically, like:

```suggestion
GitLab Code Quality Integration
###############################
```

Reply:
Fixed